sections/main-collection-product-grid.liquid

```diff
@@ -361,20 +367,28 @@
     </menu-drawer>
 
     <div class="active-facets active-facets-mobile">
-      <a href="{{ collection.url }}?sort_by={{ sort_by }}" class="active-facets__button button button--secondary js-facet-remove">{{ 'sections.collection_template.clear_all' | t }}</a>
+      <a href="{{ collection.url }}?sort_by={{ sort_by }}" class="active-facets__button js-facet-remove">
+        <span class="active-facets__button-inner button button--secondary">
+          {{ 'sections.collection_template.clear_all' | t }}
+        </span>
+      </a>
       {%- for filter in collection.filters -%}
         {%- for value in filter.active_values -%}
-          <a class="active-facets__button active-facets__button--light button button--tertiary js-facet-remove" href="{{ value.url_to_remove }}">
-            {{ value.label | escape }}
-            {% render 'icon-close-small' %}
+          <a class="active-facets__button active-facets__button--light js-facet-remove" href="{{ value.url_to_remove }}">
+            <span class="active-facets__button-inner button button--tertiary">
+              {{ value.label | escape }}
+              {% render 'icon-close-small' %}
+            </span>
           </a>
         {%- endfor -%}
 
         {% if filter.type == "price_range" %}
           {%- if filter.min_value.value != nil or filter.max_value.value != nil -%}
             <a class="active-facets__button active-facets__button--light button button--tertiary js-facet-remove" href="{{ filter.url_to_remove }}">
-              {%- if filter.min_value.value -%}{{ filter.min_value.value | money }}{%- else -%}{{ 0 | money }}{%- endif -%}-{%- if filter.max_value.value -%}{{ filter.max_value.value | money }}{%- else -%}{{ filter.range_max | money }}{%- endif -%}
-              {% render 'icon-close-small' %}
+              <span class="active-facets__button-inner button button--tertiary">
+                {%- if filter.min_value.value -%}{{ filter.min_value.value | money }}{%- else -%}{{ 0 | money }}{%- endif -%}-{%- if filter.max_value.value -%}{{ filter.max_value.value | money }}{%- else -%}{{ filter.range_max | money }}{%- endif -%}
+                {% render 'icon-close-small' %}
+              </span>
             </a>
           {%- endif -%}
```

Comment:
On Safari, the background is still scrollable: https://screenshot.click/05-51-wiacu-ueec2.mp4

Reply:
That's weird. Version?  It seems to be working as far as I can tell.

Reply:
I haven't been able to reproduce it 🤔

Reply:
🎩 Magic!
🧚

Reply:
Can't replicate anymore on my iPad either. 👍